TASK-4/CandidateReportGenerator.cs

```diff
@@ -0,0 +1,34 @@
+namespace Persons;
+
+// отсортировывает всех кандидатов по позиции по возрастанию зп
+// ***Вывод : UserId || Users Full Name || JobTittle || Salary***
+public class CandidateReportGenerator : IReportGenerator
+{
+    public IEnumerable<Candidate> ListOfCandidates;
+
+    public CandidateReportGenerator(IEnumerable<Candidate> apple)
+    {
+        ListOfCandidates = apple;
+    }
+
+    public IEnumerable<Candidate> AscendingSort()
+    {
+        return ListOfCandidates.OrderBy(c => String.Format("{0,70}{1,10}", c.DesiredPosition, c.DesiredSalary));
```

Comment:
Your sort doesn't work correctly, it doesn't sort at all

Reply:
Fixed it.